ext/web/13_message_port.js

```diff
@@ -103,6 +125,8 @@ class MessagePort extends EventTarget {
   [_id] = null;
   /** @type {boolean} */
   [_enabled] = false;
+  [_nodeWorker] = false;
+  [_closeEventDispatched] = false;
```

Comment:
We should make these two non-enumerable so that inspecting from user code doesn't show them